### ISSUE TITLE
Certificates are not added to LB when created through Rancher compose UI

### ISSUE THREAD

**Issue by aruneli** [rancher/rancher#2212]
```
Component 	Version
Rancher 	v0.40.0-rc1
Cattle 	v0.96.0
User Interface 	v0.53.0
Rancher Compose 	beta/latest

```
Created stack with below copied rancher and docker compose yml files through compose CLI. I was able to start services and verify certs were added in LB. Repeated this in UI and found certs are missing in config and also in haproxy.

docker-compose.yml

```
lbsslnew:
  restart: always
  ports:
  - 9091:80
  labels:
    io.rancher.loadbalancer.ssl.ports: '9091'
    io.rancher.loadbalancer.target.web2: def
    io.rancher.loadbalancer.target.web3: ghi
    io.rancher.loadbalancer.target.web1: abc
  tty: true
  image: rancher/load-balancer-service
  links:
  - web1:web1
  - web2:web2
  - web3:web3
  stdin_open: true
web1:
  restart: always
  tty: true
  image: sangeetha/testnewhostrouting
  stdin_open: true
web2:
  restart: always
  tty: true
  image: sangeetha/testnewhostrouting
  stdin_open: true
web3:
  restart: always
  tty: true
  image: sangeetha/testnewhostrouting
  links:
  - 'web1:'
  stdin_open: true
```
rancher-compose.yml

```
lbsslnew:
  scale: 1
  certs:
  - cert1
  - cert2
  load_balancer_config:
    description: 'lbsslnew'
    name: lbsslnew config
  default_cert: cert1
web1:
  scale: 3
web2:
  scale: 3
web3:
  scale: 3
```

**Comment by vincent99** [comment]
Not UI, I just pass the yml text to the stack create call..
@wlan0 @ibuildthecloud Is the newest version of rancher-compose that has support for certs packaged?

**Comment by ibuildthecloud** [comment]
rancher-compose-executor is using an old version of rancher-compose.  We need to sync the two now going on.

**Comment by ibuildthecloud** [comment]
@aruneli This was included in 0.40.0, can you retest?

**Comment by aruneli** [comment]
Verified in this release.
```
Component 	Version
Rancher 	v0.40.0-rc3
Cattle 	v0.97.0
User Interface 	v0.54.0
Rancher Compose 	beta/latest
```
